internal/printer/container_test.go

```diff
@@ -206,11 +209,7 @@ func Test_ContainerConfiguration(t *testing.T) {
+					Content: component.NewButton("Show Manifest", action.CreatePayload(octant.ActionGetManifest, map[string]interface{}{"host": hostOS, "image": validContainer.Image})),
```

Comment:
Since image info is fetched and shown on demand, we can add a test for when that info already exists in cache.